zephyr/src/indigo_api_callback_dut.c

```diff
@@ -1168,8 +1168,6 @@ static int assign_static_ip_handler(struct packet_wrapper *req, struct packet_wr
     reset_interface_ip(ifname);
     /* Bring up interface */
     control_interface(ifname, "up");
-    /* Set IP address with network mask */
-    strcat(buffer, "/24");
```

Comment:
Well, its not supported along with IP, but you can always set mask separately `net_if_ipv4_set_netmask`